asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py

```diff
@@ -140,7 +146,7 @@ def write_traj_visualizations(self):
             output_paths.append(output_path)
         return output_paths
 
-    def write_traj_visualization(self, traj, system, path):
+    def write_traj_visualization(self, traj, system, path, bool_static_view_only):
```

Comment:
Set a default

```suggestion
    def write_traj_visualization(self, traj, system, path, bool_static_view_only=False):
```

OR if using an attribute it shouldn't take an argument and should just use `self.static_view_only.`